queries.sql

```diff
@@ -6,3 +6,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- update the animals table by setting the species column to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon
+UPDATE animals SET species = 'unspecified' WHERE name LIKE '%mon';
```

Comment:
- [ ] Here, you set the species to `unspecified`, however, you are required to set it to `digimon`, kindly fix this

Reply:
@Ghiftee done with this change